manager-bundle/bin/contao-console

```diff
@@ -31,12 +31,8 @@ if (file_exists(__DIR__.'/../autoload.php')) {
 $loader = require $projectDir.'/vendor/autoload.php';
 
 $input = new ArgvInput();
-$env = $input->getParameterOption(['--env', '-e'], getenv('SYMFONY_ENV') ?: 'prod');
+$env = $input->getParameterOption(['--env', '-e'], null);
```

Comment:
This seems strange to me, as the interface expects a boolean value as second argument.